libs/libc/misc/lib_execinfo.c

```diff
@@ -88,3 +88,38 @@ void backtrace_symbols_fd(FAR void *const *buffer, int size, int fd)
       dprintf(fd, "%pS\n", buffer[i]);
     }
 }
+
+/****************************************************************************
+ * Name: backtrace_format
+ *
+ * Description:
+ *  Format a backtrace into a buffer for dumping.
+ *
+ ****************************************************************************/
+
+FAR const char *backtrace_format(FAR char *buffer, int size,
+                                 FAR void *backtrace[], int depth)
+{
+  FAR const char *format = "%0*p ";
```

Comment:
```suggestion
  FAR const char *format = "%0*p";
```

Reply:
The space is to separate the backtraces. Is it acceptable for backtrace squeezed together?

Reply:
Aha, I see your point. By specifying the width with one space larger, it will take care of it. I'll update the code.